client/my-sites/customer-home/cards/notices/celebrate-site-creation-v2/index.jsx

```diff
@@ -18,7 +19,7 @@ const CelebrateSiteCreation = () => {
 			description={ translate(
 				"Next, we'll guide you through setting up and launching your site."
 			) }
-			noticeId="site-created"
+			noticeId={ NOTICE_CELEBRATE_SITE_CREATION }
```

Comment:
🤔 This will likely break the logic we have in the API for handling the legacy card UIDs used in the celebratory notices. Looking into a server path so we can safely land this.

Reply:
D43588-code